test-kit/src/test/scala/rating/glicko/GlickoCalculatorWithColorAdvantageTest.scala

```diff
@@ -0,0 +1,144 @@
+package chess.rating.glicko
```

Comment:
I think we should see some `assertCloseTo` testing in this file as well, as to see the impact of color advantage. `>` is useful but it could mean a lot of things.